src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  gap: $spaceBetweenDates;
+  flex-wrap: wrap;
+  margin-top: 30px;
+  width: ($paddings * 2) + ($widthsDates * 7) + ($spaceBetweenDates * 6);
+  padding: $paddings;
```

Comment:
Notice - you are adding paddings to the calendar twice - that causes extra padding above 10px

<img width="782" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/1983879/8b3489ce-fb8c-44db-9c61-e89013d13992">